docs/en/ingest-management/elastic-agent/otel-agent.asciidoc

```diff
@@ -0,0 +1,145 @@
+[[otel-agent]]
+= Run {agent} as an OTel Collector
+
+preview::[]
+
+The link:https://opentelemetry.io/docs/collector/[OpenTelemetry Collector] is a vendor-neutral way to receive, process, and export telemetry data. {agent} includes an embedded OTel Collector, enabling you to instrument your applications and infrastructure once, and send data to multiple vendors and backends. 
+
+When you run {agent} in `otel` mode it supports the standard OTel Collector configuration format that defines a set of receivers, processors, exporters, and connectors. Logs, metrics, and traces can be ingested using OpenTelemetry data formats.
+
+The following steps will help you to get set up quickly using {agent} as an OTel Collector. Note that all steps are run with `sudo` as it's generally recommended to run {agent} with administrative privileges.
+
+[discrete]
+== Run {agent} in OTel mode
+
+For simplicity, these steps demonstrate OTel mode using standalone {agent}. The install steps for {fleet}-managed {agent} are very similar. Refer to the <<elastic-agent-installation,Install {agents}>> section for the full set of requirements and steps for any install scenario.
+
+. On your host, download and extract the {agent} installation package. 
++
+--
+// tag::install-elastic-agent[]
+
+include::{ingest-docs-root}/docs/en/ingest-management/tab-widgets/download-widget.asciidoc[]
+
+// end::install-elastic-agent[]
+--
++
+The commands shown are for AMD platforms, but ARM packages are also available.
+Refer to the {agent} https://www.elastic.co/downloads/elastic-agent[downloads page]
+for the full list of available packages.
+
+. Change into the agent package directory and run {agent} with the `otel` subcommand and the `-h` option to view the command syntax and available parameters:
++
+[source,shell]
+--
+cd <agent-package-name>
+sudo ./elastic-agent otel -h
+--
++
+TIP: You can also refer to the <<elastic-agent-otel-command,{agent} OTel>> command documentation for the full command syntax.
+
+. View or update the OTel configuration file.
++
+A basic OTel configuration file named `otel.yml` is included in the {agent} package. You can view or update it using any text editor.
++
+[source,yaml]
+--
+receivers:
+  filelog:
+    include: [ /var/log/system.log ]
+    start_at: beginning
+
+processors:
+  resource:
+    attributes:
+    - key: service.name
+      action: insert
+      value: elastic-otel-test
+
+exporters:
+  debug:
+    verbosity: detailed
+    sampling_initial: 10000
```

Comment:
this will print the first 10000 lines to the console but then only print every 10000th line, so we may want to pick different numbers here like 10 and 10 or something and then add a note about it

Reply:
Why would we want to have debug exporter's output sampled at all?

Note that sampling in the Debug exporter will be disabled by default in the next version of the collector (v0.103.0), since this PR has been merged:
- https://github.com/open-telemetry/opentelemetry-collector/pull/10250

I believe we should remove the sampling configuration options `sampling_initial`, `sampling_thereafter` from our config and rely on the default configuration of the component.

Reply:
I think it's easiest for customers if the docs exactly match the [sample config file](https://github.com/elastic/elastic-agent/blob/main/otel.yml) in the elastic agent packages, so for now I've left the `sampling_initial` and `sampling_thereafter` settings as is.

When these are removed I'd really appreciate if someone can tag me on the PR and I'll update the docs accordingly.